src/window_item.cpp

```diff
@@ -126,8 +126,8 @@ void Window_Item::DrawItem(int index) {
 		std::stringstream ss;
 		ss << number;
 		Font::SystemColor color = enabled ? Font::ColorDefault : Font::ColorDisabled;
-		contents->TextDraw(rect.x + rect.width - 28, rect.y, color, "x");
-		contents->TextDraw(rect.x + rect.width - 6, rect.y, color, ss.str(), Text::AlignRight);
+		contents->TextDraw(rect.x + rect.width - 24, rect.y, color, "x");
+		contents->TextDraw(rect.x + rect.width, rect.y, color, ss.str(), Text::AlignRight);
```

Comment:
All versions of RPG_RT I checked use `:` not `x`

Change this to a single TextDraw call with `fmt::format(":{:3d}", number)`

Reply:
About which scene are we talking here?

![Unbenannt](https://user-images.githubusercontent.com/1331889/94325459-c0a25080-ff9e-11ea-983e-fad9c8794cf5.png)

Reply:
I tried 2k3 dynrpg, 2ke steam, and 2k3e steam

Reply:
Changed.

Reply:
We can argue a while about one char. For menu scene implementations we have some freedom: We can do enhancements. the marker for "times you have something" is in all languages I know the "X" and not a ":". Not 100% sure about Japanese. They have lots of counter kanji

Reply:
I personally prefer `x` over `:` but I came across a case where `x` looks ugly: If an item uses the full item string length then it looks like the `x` belongs to the item name. So I went for @fmatthew5876's suggestion to use `:` instead of `x`.

Reply:
Yet another example of a custom EasyRPG database chunk